utils.go

```diff
@@ -210,7 +209,7 @@ func getRootlessStorageOpts(rootlessUID int) (StoreOptions, error) {
 	opts.RootlessStoragePath = opts.GraphRoot
 	if path, err := exec.LookPath("fuse-overlayfs"); err == nil {
 		opts.GraphDriverName = "overlay"
-		opts.GraphDriverOptions = []string{fmt.Sprintf("overlay.mount_program=%s", path)}
+		opts.GraphDriverOptions = append(opts.GraphDriverOptions, fmt.Sprintf("overlay.mount_program=%s", path))
```

Comment:
Does this break when the system `opts.GraphDriverName` is not "overlay"?

Reply:
I am not sure.  I think this could cause a few issues, which is why I stated that we need to be careful, even if this is the correct thing to do.  I guess if we do the else option, we should just clear these fields.

Reply:
Could we copy these options only when `opts.GraphDriver` is also set to `overlay`?

Reply:
Fixed